torch/_dynamo/variables/user_defined.py

```diff
@@ -845,11 +845,30 @@ def odict_getitem(self, tx, key):
 
         # TODO this should probably be merged with the dict handling
```

Comment:
Wouldn't it be better to refactor OrderedDict into a subclass of `ConstantDictVariable`?